README.md

```diff
@@ -131,19 +131,6 @@ const layout = node.computeLayout();
 console.log(layout.width, layout.height);
 ```
 
-## Contributing
```

Comment:
Should we add a reference to the CONTRIBUTING.md here or do you want to keep it completely out of the README?

Reply:
I don't think it's worth it. This will let us include the README as the crate-level docs, which is very convenient.